MAUI/Scheduler/header.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: Header in .NET MAUI Scheduler control | Syncfusion
+description: Learn here all about to customize header height, date format and appearance of Syncfusion .NET MAUI Scheduler (SfScheduler) control and more. 
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Header in .NET MAUI Scheduler (SfScheduler)
+
+You can learn about SfScheduler's header height, date format, and appearance support of the SfScheduler.
```

Comment:
You can learn about SfScheduler's header height, date format, and appearance support.

Please highlight the control names